scripts/copy_meta_from_json.py

```diff
@@ -0,0 +1,69 @@
+import re, json
```

Comment:
Can you add some description at the top as to what this script does, how it is currently used (or not, if it was a one-time thing), how you can execute it?

Reply:
yea sure!